File: scratch/db_check.js

const { Pool } = require('pg');

const DATABASE_URL = "postgresql://postgres.yumredpotusegemxudft:TorreControl2026!@aws-1-sa-east-1.pooler.supabase.com:6543/postgres";

const pool = new Pool({
  connectionString: DATABASE_URL,
  ssl: { rejectUnauthorized: false }
});

async function checkSnapshots() {
  try {
    const res = await pool.query("SELECT * FROM evento_kpi_snapshots WHERE date = '2026-05-11'");
    console.log('Snapshots for 2026-05-11:');
    console.log(JSON.stringify(res.rows, null, 2));
    await pool.end();
  } catch (err) {
    console.error('Error:', err);
  }
}

checkSnapshots();

File: scratch/db_check.py

import psycopg2
import os

DATABASE_URL = "postgresql://postgres.yumredpotusegemxudft:TorreControl2026!@aws-1-sa-east-1.pooler.supabase.com:6543/postgres"

def check_snapshots():
    try:
        conn = psycopg2.connect(DATABASE_URL)
        cur = conn.cursor()
        cur.execute("SELECT * FROM evento_kpi_snapshots WHERE date = '2026-05-11'")
        rows = cur.fetchall()
        
        colnames = [desc[0] for desc in cur.description]
        print(f"Column Names: {colnames}")
        
        if not rows:
            print("No snapshot found for 2026-05-11")
        else:
            for row in rows:
                print(dict(zip(colnames, row)))
        
        cur.close()
        conn.close()
    except Exception as e:
        print(f"Error: {e}")

check_snapshots()
